apps/omg_watcher/lib/omg_watcher/exit_processor/standard_exit.ex

```diff
@@ -79,10 +79,13 @@ defmodule OMG.Watcher.ExitProcessor.StandardExit do
     exits_invalid_by_ife = get_invalid_exits_based_on_ifes(active_exits, tx_appendix)
     invalid_exits = active_exits |> Map.take(invalid_exit_positions) |> Enum.concat(exits_invalid_by_ife) |> Enum.uniq()
 
+    ethereum_block_time_seconds = OMG.Eth.Configuration.ethereum_block_time_seconds()
     # get exits which are still invalid and after the SLA margin
     late_invalid_exits =
       invalid_exits
-      |> Enum.filter(fn {_, %ExitInfo{eth_height: eth_height}} -> eth_height + sla_margin <= eth_height_now end)
+      |> Enum.filter(fn {_, %ExitInfo{eth_height: eth_height}} ->
+        eth_height + sla_seconds / ethereum_block_time_seconds <= eth_height_now
```

Comment:
Is this a good way or should the timestamp of each exit be stored in the DB and retrieved here to check if sla_margin is elapsed

Reply:
I think this is fine for this PR since it's still the existing behaviour, but could you maybe raise this discussion separately in the engineering channel?

Reply:
will change the logic to  `exit_timestamp + sla_seconds <=  time_now` when #1495 is merged